### ISSUE TITLE
retention.container.success_duration being ignored

### ISSUE THREAD

**Issue by srbry** [concourse/concourse#492]
When setting `retention.container.success_duration` on my concourse bosh release, it lets me set the variables to `1s` but running `fly -t target cs` shows that successful jobs have containers set with a 5m validity

**Comment by vito** [comment]
Are you sure you're not seeing containers that are either before you set the property or are actually for builds that are in-flight?

**Comment by srbry** [comment]
I am quite sure. My bosh config for the ATC is:
```
 - name: atc
    release: concourse
    properties:
      publicly_viewable: true
      external_url: example.com
      basic_auth_username: user
      basic_auth_password: password
      postgresql_database: atc
      old_resource_grace_period: 1m
      retention:
        container:
          success_duration: 1s
          failure_duration: 10m
```

When I run `fly -t targer cs` I get:
```
handle                                ttl       validity  worker                                pipeline        job         build #  build id    type    name                            attempt
0397bf7c-0ad8-4c0b-7f3e-cc07c5b2974e  00:00:42  00:05:00  afa3aa6a-74a0-4bba-8b16-5e3fac1cb34f  cf-monitoring   job-name    2368     18149       check   test                            n/a
039b6244-33e7-414b-46c7-a5b8cbdc30ce  00:04:04  00:05:00  afa3aa6a-74a0-4bba-8b16-5e3fac1cb34f  cf-monitoring   job-name    795      18170       get     every-3-minutes                 n/a
03f7d75b-dfd5-420b-7442-638ac378badf  00:04:34  00:05:00  afa3aa6a-74a0-4bba-8b16-5e3fac1cb34f  cf-monitoring   job-name    2373     18183       check   test                            n/a
041e2529-ee56-4f65-42f7-789d71454e8f  00:04:28  00:05:00  e2401af7-52f7-4b49-9790-cc5dbace96a6  cf-monitoring   job-name    784      18178       check   smoke-test                      n/a
0567681f-cd15-4a61-47fc-5cbf3e24ab6e  00:00:42  00:05:00  afa3aa6a-74a0-4bba-8b16-5e3fac1cb34f  cf-monitoring   job-name    2372     18152       get     test                            n/a
05ad1d46-dab6-4409-747f-6c0fcca9cf54  00:03:34  00:05:00  e2401af7-52f7-4b49-9790-cc5dbace96a6  cf-monitoring   job-name    2371     18172       get     pipeline                        n/a
05f682e6-c168-4725-6d61-27a7f0022b81  00:04:31  00:05:00  afa3aa6a-74a0-4bba-8b16-5e3fac1cb34f  cf-monitoring   job-name    2379     18184       get     every-minute                    n/a
05ff792c-1467-4452-7144-741b77abc316  00:01:01  00:05:00  e2401af7-52f7-4b49-9790-cc5dbace96a6  cf-monitoring   job-name    794      18147       get     pipeline                        n/a
065aeb4d-ba11-45f7-7f4f-079f671ae22e  00:02:32  00:05:00  e2401af7-52f7-4b49-9790-cc5dbace96a6  cf-monitoring   job-name    2374     18168       get     every-minute                    n/a
07f4dc44-fa0b-4e9f-51ed-4a16f81fbae8  00:04:26  00:05:00  e2401af7-52f7-4b49-9790-cc5dbace96a6  cf-monitoring   job-name    2374     18181       get     resource                        n/a
0844be44-0dbd-4d0f-7a20-9986887fae9e  00:03:33  00:05:00  afa3aa6a-74a0-4bba-8b16-5e3fac1cb34f  cf-monitoring   job-name    2373     18171       get     pipeline                        n/a
09974797-52f5-452d-47ba-7f1d197f468e  00:00:38  00:05:00  afa3aa6a-74a0-4bba-8b16-5e3fac1cb34f  cf-monitoring   job-name    2375     18151       get     every-minute                    n/a
0bf98e32-aa8c-471d-64ee-d43075aa46ed  00:00:43  00:05:00  afa3aa6a-74a0-4bba-8b16-5e3fac1cb34f  cf-monitoring   job-name    2372     18153       get     test                            n/a
0c31e304-183c-45c0-7d36-75fa4da75de5  00:01:26  00:05:00  afa3aa6a-74a0-4bba-8b16-5e3fac1cb34f  cf-monitoring   job-name    783      18156       get     resource                        n/a
```

All the jobs complete in under a minute so having the TTLs as above would suggest that they are not still in-flight to me.

**Comment by vito** [comment]
Container retention policies are no longer a thing with 1.4.0/1.4.1:

https://concourse.ci/downloads.html#v140

**Comment by FayeColeman** [comment]
Before you place any investment with a broker, ensure that you do your proper research so you don't end up sending scammers your hard-earned money. I suffered from this after I sent a scammer I perceived to be a legitimate broker, but it later turned out he was a scammer. But I am happy I got introduced to RecoveryHacker101, because indeed I wouldn't have gotten my funds if not for their expert services in tracing stolen crypto funds. I eventually got my money back and, trust me, I am not investing anymore. I can remember the sudden pain I got after I realized the broker was a scammer. That's why I have taken my time to post this comment here to anyone in need of a crypto recovery expert can simply reach them at (recoveryhacker101@gmailcom). I vouch for their services 100%.
